client/src/Components/HomePage/MenuItemCategories/MenuItemCategory/CategorizedMenuItem/CategorizedMenuItem.jsx

```diff
@@ -12,7 +12,7 @@ import {
   loginFormVisibilityChanged,
 } from "../../../../../slices/loginSlice";
 
-export default function CategorizedMenuItem(props) {
+export default function CategorizedMenuItem({ details }) {
```

Comment:
I would accept here name and price, it's not clear what is details.
Think about the component as about something complete and reusable, it should be clear how to use it